packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -16,6 +16,67 @@ import 'viewport_offset.dart';
 // CORE TYPES FOR SLIVERS
 // The RenderSliver base class and its helper types.
 
+/// Called to get the item extent by the index of item.
+typedef ItemExtentGetter = double Function(int index, SliverLayoutDimensions dimensions);
+
+/// Holds the relation dimensions of the [RenderSliver] during layout.
```

Comment:
```suggestion
/// Relates the dimensions of the [RenderSliver] during layout.
///
/// Used by...
```

Can add a reference to the item extent callback from sliver list / list view?